jekyll/_cci2/managing-api-tokens.md

```diff
@@ -9,75 +9,95 @@ version:
 - Server v2.x
 ---
 
-To use the CircleCI API
-or view details about a build,
-you will need API tokens with the appropriate permissions.
-This document describes the types of API tokens,
-as well as how to create and delete them.
+To use the CircleCI API or view details about a build, you will need API tokens
+with the appropriate permissions. This document describes the types of API
+tokens, as well as how to create and delete them.
 
 ## Overview
 
-There are two types of API tokens
-you can create within CircleCI.
+There are two types of API tokens you can create within CircleCI.
 
   1. **Personal**:
   These tokens are used to interact with the CircleCI API
   and grant full read and write permissions.
   2. **Project**:
   These tokens allow you to read/write information for specific projects.
   Project tokens have three scope options: _Status_, _Read Only_, and _Admin_.
-      - _Status_ tokens grant read access to the project's build statuses.
-      Useful for [embedding status badges]({{ site.baseurl }}/2.0/status-badges/).
-      - _Read Only_ tokens grant read only access to the project's API.
-      - _Admin_ tokens grant read and write access for the project's API.
+    - _Status_ tokens grant read access to the project's build statuses.
+    Useful for [embedding status badges]({{ site.baseurl }}/2.0/status-badges/).
+    - _Read Only_ tokens grant read only access to the project's API.
+    - _Admin_ tokens grant read and write access for the project's API.
 
-**Note**:
-API tokens cannot be modifed
-after they have been created.
-The only way to change existing tokens
-is to delete and recreate them.
+**Note**: API tokens cannot be modified after they have been created. The only
+way to change existing tokens is to delete and recreate them.
```

Comment:
@teesloane -- would we want to say here that `The only way to change existing tokens is to delete and recreate them, also known as token rotation`, to make it clear that the token rotation steps will do this job?

Reply:
Good call. Updated 👍 !